### PR TITLE
Clean up SQL for Postgres

### PR DESCRIPTION
**Pull request by hkd987**
Resolve issues found, https://github.com/hwchase17/langchainjs/issues/474. Where Postgres schema is not respected.

### CODE REVIEW

**Review by agola11 — COMMENTED** [review]
Thanks! Were you able to test this?

File: langchain/src/util/sql_utils.ts [hunk 1]
```diff
@@ -109,20 +109,11 @@ export const getTableAndColumnsName = async (
...
-      "        JOIN information_schema.columns c\n" +
-      "             ON t.table_name = c.table_name\n" +
-      "WHERE\n" +
-      "        t.table_schema = 'public'\n" +
-      "ORDER BY\n" +
-      "    t.table_name,\n" +
-      "    c.ordinal_position;";
-
+    const schema = appDataSource.options?.schema ?? "public";
+    sql = `SELECT t.table_name, c.* FROM information_schema.tables t
```
[the remark this answers is not in the document]
Comment (hkd987) [reply langchain/src/util/sql_utils.ts:113]: Sure, I’ll update in the morning.

[the remark this answers is not in the document]
Comment (hkd987) [reply langchain/src/util/sql_utils.ts]: @agola11 -> Updated to match how it looked before.

**Review by agola11 — APPROVED** [review]
thanks, looks good!

**Pull request merged**